src/nwb_datajoint/spikesorting/sortingview.py

```diff
@@ -153,44 +170,30 @@ def insert_manual_curation(self, key: dict, description=''):
         description: str
             optional description of curated sort
         """
+        print(key)
         workspace_uri = (SortingviewWorkspace & key).fetch1('workspace_uri')
         workspace = sv.load_workspace(workspace_uri=workspace_uri)
 
         sortingview_sorting_id = (SortingviewWorkspace & key).fetch1(
             'sortingview_sorting_id')
-        manually_curated_sorting = workspace.get_curated_sorting_extractor(
-            sorting_id=sortingview_sorting_id)
 
         # get the labels and remove the non-primary merged units
         labels = workspace.get_sorting_curation(
             sorting_id=sortingview_sorting_id)
-        # turn labels to list of str, only including accepted units.
 
-        unit_labels = labels['labelsByUnit']
-        unit_ids = [unit_id for unit_id in unit_labels]
-
-        # load the metrics
-        from .spikesorting_curation import QualityMetrics
-        metrics_path = (QualityMetrics & {'nwb_file_name': key['nwb_file_name'],
-                                          'recording_id': key['recording_id'],
-                                          'waveform_params_name': key['waveform_params_name'],
-                                          'metric_params_name': key['metric_params_name'],
-                                          'sorting_id': key['parent_sorting_id']}).fetch1('quality_metrics_path')
-        with open(metrics_path) as f:
-            quality_metrics = json.load(f)
-
-        # remove non-primary merged units
-        clusters_merged = bool(labels['mergeGroups'])
-        if clusters_merged:
-            for m in labels['mergeGroups']:
-                for merged_unit in m[1:]:
-                    unit_ids.remove(merged_unit)
-                    del labels['labelsByUnit'][merged_unit]
-            # if we've merged we have to recompute metrics
-            metrics = None
+        # turn labels to list of str, only including accepted units.
+        if bool(labels['mergeGroups']):
+            # clusters were merged, so we empty out metrics
+            metrics = {}
         else:
-            metrics = quality_metrics
+            # get the metrics from the parent curation
+            metrics = (Curation & key).fetch1('metrics')
 
         # insert this curation into the  Table
-        return self.insert_curation(key, parent_curation_id=key['parent_curation_id'], labels=labels['labelsByUnit'],
-                                    merge_groups=labels['mergeGroups'], metrics=metrics, description='manually curated')
+        return Curation.insert_curation(
+            key,
+            parent_curation_id=key['curation_id'],
+            labels=labels['labelsByUnit'],
+            merge_groups=labels['mergeGroups'],
+            metrics=metrics,
+            description='manually curated')
```

Comment:
seems setting the `description` argument to this method doesn't do anything at the moment because it is always set to `manually curated`. see above comment.